Sources/SwiftDocCUtilities/Action/Actions/Convert/ConvertAction.swift

```diff
@@ -221,12 +214,22 @@ public struct ConvertAction: Action, RecreatingContext {
                 allowArbitraryCatalogDirectories: allowArbitraryCatalogDirectories
             )
         } else {
-            self.context.externalMetadata.isGeneratedBundle = true
+            configuration.externalMetadata.isGeneratedBundle = true
             dataProvider = GeneratedDataProvider(symbolGraphDataLoader: { url in
                 fileManager.contents(atPath: url.path)
             })
         }
         
+        if let outOfProcessResolver {
+            configuration.externalDocumentationConfiguration.sources[outOfProcessResolver.bundleIdentifier] = outOfProcessResolver
+            configuration.externalDocumentationConfiguration.globalSymbolResolver = outOfProcessResolver
+        }
+        
+        (context as _DeprecatedConfigurationSetAccess?)?.configuration = configuration
```

Comment:
This silences a deprecation warning. We can't get rid of this line without causing breaking changes in case someone passed a context to the ConvertAction initializer (the default value for that parameter is `nil`).